providers-sdk/v1/lr/cli/cmd/markdown.go

```diff
@@ -175,7 +171,7 @@ func (l *lrSchemaRenderer) renderToc(packName string, description string, resour
 
 	// render content
 	builder.WriteString("# Mondoo " + packName + " Resource Pack Reference\n\n")
-	builder.WriteString("In this pack:\n\n")
+	builder.WriteString("Resources included in this pack:\n\n")
```

Comment:
@misterpantz can you let me know if this works for you. This goes on the main resource page pages like https://mondoo.com/docs/mql/resources/gcp-pack/